pkg/metric/dsplugin.go

```diff
@@ -0,0 +1,24 @@
+package metric
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/graphbound/graphbound/pkg/httpds"
+)
+
+func NewHTTPDSPlugin() httpds.Plugin {
+	return func(c *httpds.Context) {
+		start := time.Now()
+		reqHost := c.Request.URL.Host
+		reqPath := c.FullPath()
```

Comment:
Doesn't the request have a path? Why don't we need a full path here?

Reply:
Full path means the path with templated parameters. Similar to what happens in gin.
```go
fmt.Println(c.Request.URL.Path) // /users/12345
fmt.Println(c.FullPath()) // /users/:id
```